int32 currentMowingPath
int32 currentMowingArea
int32 currentMowingPathIndex
string currentMowingPlanDigest
float32 currentMowingAngleIncrementSum